src/Annotator.scss

```diff
@@ -291,20 +210,23 @@
             margin-right: 0;
         }
     }
+}
 
-    .delete-comment-btn {
-        position: absolute;
-        right: 0;
-        top: 0;
+ul .ba-annotation-list-item {
```

Comment:
Please try to avoid tag names in CSS. Should this be combined with the other style rules above?